### PR TITLE
feat(GUI): enlarge reflash button on success page

### PR DESCRIPTION
**Pull request by Shou**
We change the `Flash Another` button in the top-right corner of the
success screen to be bigger, matching the banner buttons' size.

Changelog-Entry: Enlarge Flash Another button on success page.

### CODE REVIEW

**Comment by Shou** [comment]
image

File: lib/gui/pages/finish/styles/_finish.scss [hunk 1]
```diff
@@ -26,7 +26,9 @@
 }
 
 .page-finish .button-primary {
-  min-width: $btn-min-width;
```
Comment (jviotti) [inline lib/gui/pages/finish/styles/_finish.scss]: We shouldn't just stop of using this variable. We either update it, so all buttons have the same min width, or adjust the banner's buttons to match the application.

Comment (lurch) [inline lib/gui/pages/finish/styles/_finish.scss]: I think if we made _all_ the buttons have this same minimum width, then the three buttons of the main UI wouldn't fit into the window?

**Comment by lurch** [comment]
@Shou just for comparison purposes, could you also post a screenshot of how this change looks when the offline banner is displayed?

File: lib/gui/pages/finish/styles/_finish.scss (hunk 1 above)
Comment (jviotti) [inline lib/gui/pages/finish/styles/_finish.scss]: I'm also happy with a new variable called `$btn-large-min-width` or something like that. Basically my point is to try to not make many different exceptions to something like a button width on different parts of the app.

**Comment by Shou** [comment]
@konmouz so the size is matched exactly now, but it still appears bigger, probably because it's lurid compared to the other buttons. 
image

@lurch here's the offline banner.
image

(the discrepancies of the two are quite clear here, I wish we synced their similarities somehow...)

**Comment by lurch** [comment]
Thanks for the screenshots @Shou !

> the discrepancies of the two are quite clear here, I wish we synced their similarities somehow...

I believe @jviotti is planning to publish Etcher v1.1 soon, so maybe that'd be a good opportunity to update the offline-banner to better match the (current) online-banner?

**Pull request closed without merging**